bibliopixel/drivers/SPI/APA102.py

```diff
@@ -0,0 +1,27 @@
+from ... import gamma
+from .. channel_order import ChannelOrder
+from . base import SPIBase
+
+
+class APA102(SPIBase):
+    """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
+
+    def __init__(self, num, gamma=gamma.APA102, **kwargs):
+        super().__init__(num, gamma=gamma, **kwargs)
+
+        # APA102 requires latch bytes at the end
+        self._latchBytes = (int(num // 64.0) + 1)
```

Comment:
Good point, I had a bunch of failures do to having / but not the int() so I was just adding // everywhere I knew it was supposed to be int. Removing the int()